examples/scenarios/syslog.md

```diff
@@ -0,0 +1,40 @@
+# Syslog
+
+Once you have Stanza installed and running from the [quickstart guide](./README.md#quick-start), you can follow these steps to configure Syslog monitoring.
+
+## Prerequisites
+
+It may be necessary to add an inbound firewall rule on Windows. To do this:
```

Comment:
I like the windows specific examples, it might be nice to also mention linux (firewalld and ufw are popular firewalls on linux). 

firewalld
```
firewall-cmd --permanent --add-port=514/udp
firewall-cmd --reload
```
ufw
```
ufw allow 514
```

Reply:
added those linux firewall commands